refresh/runner.go

```diff
@@ -17,7 +18,10 @@ func (m *Manager) runner() {
 			// kill the previous command
 			pid := cmd.Process.Pid
 			m.Logger.Success("Stopping: PID %d", pid)
-			cmd.Process.Kill()
+			if err := cmd.Process.Signal(syscall.SIGINT); err != nil {
```

Comment:
On windows, it should return an error. On error, this defaults to Process.Kill(). Unfortunately i do not have a windows box to test.

from exec_windows.go

func (p *Process) signal(sig Signal) error {
        handle := atomic.LoadUintptr(&p.handle)
        if handle == uintptr(syscall.InvalidHandle) {
                return syscall.EINVAL
        }
        if p.done() {
                return errors.New("os: process already finished")
        }
        if sig == Kill {
                return terminateProcess(p.Pid, 1)
        }
        // TODO(rsc): Handle Interrupt too?
        **return syscall.Errno(syscall.EWINDOWS)**
}